### PR TITLE
Created Portfolio project

### PR DESCRIPTION
**Pull request by ksigei**
### Portfolio: setup and mobile version skeleton
**In this milestone, I was able to:**

- Set up a new repository and prepare it for development using best practices ( linters).
- Build the first 2 sections of the mobile website using the template I chose in the previous step.
          1.        the toolbar (or header):
          2.        the headline section (right after the header)
- Layout the elements on the page using Flexbox in all 2 sections.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="header-container">
+        <div class="header-img-left">
+            <img src="images/image_geometry_1.png" alt="image">
+        </div>
+        <div class="header-nav-lg">
+            <img src="images/ic_menu.jpg" alt="image">
+        </div>
```
Comment (BenMukebo) [inline index.html]: - [ ] Please, wrap these `2 divs` inside a `header` tag, this should represent the header section, by this, the div containing the humburger should be replaced with a `nav` tag instead.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            <h1>Kiprono Japhet</h1>
+        </div>
+        <div class="header-about">
+            <p class="aboutme">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+                  If you like what you see and have a project you need coded, don’t hesitate and contact me.
+            </p>
+        </div>
+        <!-- <div class="header-btn">
+            <button>Start collaboration</button>
+        </div> -->
```
Comment (BenMukebo) [inline index.html]: - [ ] Please, don't leave unused code as commented in your codebase, removing them will make your code professional and clean, let's go and fix it 😄

File: style.css [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+}
+
+.header-my-name h1 {
+  font-family: Playfair Display, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 50px;
+  line-height: 100%;
+
+  /* or 50px */
```
Comment (BenMukebo) [inline style.css]: - [ ] Please, do the same for this line too 😄

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi **@Kip Kiprono,**

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!

### To highlight 💪🏻

- No linter errors. ✔️ 
- Github flow was correctly used.  ✔️   
- Good commit messages ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

- [ ] Please, Write a meaningful title for your pull request and always add a short summary of changes introduced in the pull request. you can check the  [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

Cheers :clinking_glasses:  and Happy coding!!! :dancers:

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by keroloussamy — APPROVED** [review]
Hi @ksigei, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap

## Optional suggestions

- [ ] Please remember to name your branch with a descriptive name instead of `milestone1`, e.g. 'created toolbar and headline' or 'added-toolbar-and-headline-sections'

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**